### ISSUE TITLE
Way to `add` all previously added files/folders

### ISSUE THREAD

**Issue by pvinis** [twpayne/chezmoi#690]
## Is your feature request related to a problem? Please describe.

Since chezmoi knows the files that have been `add`ed to it, it would be great to have a command like `chezmoi add-all` or something, that will add all added files.

Example:
- new chezmoi
- add .zshrc
- add .config/git
- add .asdfrc
- commit all that stuff
Then later, after a week, I have done some changes in some of those files. I'd love to do `chezmoi add-all` and chezmoi would run the add command for .zshrc, .config/git and .asdfrc.

Does that make sense?

## Describe the solution you'd like

To have a command to do this automatically.

## Describe alternatives you've considered

I was thinking of making a script for me to do that, but an "official" command might be easier for others too.

**Comment by twpayne** [comment]
Thanks for the suggestion. The trouble with `chezmoi add` is that it will overwrite files in the source state, which means that it will overwrite templates in the source state too. This would make `add-all` quite a dangerous command to run. Is there a way around this?

**Comment by pvinis** [comment]
hmm I hadn't thought of that. I think that if I have a .somerc and it's a template, so dot_somerc.tmpl, if that command would just take the .somerc and put it as dot_somerc.tmpl in my chezmoi for and overwrite the old, it might still be easy to "fix" the template stuff using git and just discarding some of the changes.

A better but a bit more hard to implement way would be this:
- get the list of chezmoi files including tmpl
- chezmoi add all, but for the tmpl files do a special thing.
that special thing would be to get the old file (one in chezmoi dir), get the value of the 1pass key for example, and get the new file (one from home dir) and do a replacement in there. Find the value that we got from 1pass, and replace it with the tmpl part that we got the value with.

That doesn't solve the case where we have multiple machines and the Vue depends on that, but it's something.
